ANMN/burst_averaged_product/burst_average.py

```diff
@@ -25,15 +25,13 @@ def get_input_file_rel_path(input_netcdf_file_path):
     find the relative path hierarchy of an input FV01 file. The value will be
     used in a gatt of the burst netcdf file
     """
-    file_path = MooringFileClassifier.dest_path(input_netcdf_file_path)
-    return file_path
+    return MooringFileClassifier.dest_path(input_netcdf_file_path)
 
 def get_burst_duration(netcdf_file_obj):
     """
     returns the burst duration as float defined in the NetCDF file if exists
     """
-    burst_duration  = getattr(netcdf_file_obj, 'instrument_burst_duration', 60.0)
-    return burst_duration
+    return getattr(netcdf_file_obj, 'instrument_burst_duration', 60.0)
```

Comment:
This default value is dodgy to me. Especially since this is being used later to identify bursts.

There is/will be cases when it is not 60sec and clearly this can be inferred from the input data itself.